cob_twist_controller/include/cob_twist_controller/cob_twist_controller.h

```diff
@@ -78,10 +79,11 @@ class CobTwistController
     ros::Subscriber twist_stamped_sub;
     ros::Subscriber base_sub;
     ros::Publisher vel_pub;
+    ros::Publisher pos_pub;
     ros::Publisher base_vel_pub;
+    ros::Publisher pub;
```

Comment:
I think the vel_pub, pos_pub and pub are not required anymore because they are rplaced by the InterfaceType->process_result...